core/src/main/java/org/mobilitydata/gtfsvalidator/notice/DuplicatedColumnNotice.java

```diff
@@ -18,7 +18,11 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** The input file CSV header has the same column name repeated. */
+/**
+ * The input file CSV header has the same column name repeated.
+ *
+ * <p>Severity: {@code SeverityLevel.WARNING}
```

Comment:
IMHO, this is an ERROR. The consumer would not know which column to rely on, and this would make the dataset unusable.
Is there a similar rule for files that get provided twice or more?

Registered in Discrepancies Specification vs Validator MobilityData/gtfs-tasks#87

Reply:
@timmillet you link to non public ressources, is that wanted? :)

Reply:
@timMillet 

> Is there a similar rule for files that get provided twice or more?

We actually do not have this rule implemented. @barbeau should we open an issue in the validator now on this matter?

Reply:
@timMillet we do not have this rule implemented. But from my understanding having such a situation is not possible (@barbeau  correctly me if I'm wrong please). One cannot have two .txt files having the same name on the same repository path: one of the files would be name "agency-copy.txt", which would be flagged as UnknownFileError.

Reply:
>Is there a similar rule for files that get provided twice or more?
> One cannot have two .txt files having the same name on the same repository path: one of the files would be name "agency-copy.txt", which would be flagged as UnknownFileError.

From some quick Googling it seems like it might be possible to intentionally include two files with the same file name within a zip file, but it would take some intentional programming jujitsu as most normal zip tools won't allow it. I'd be inclined to ignore this case until we encounter it in practice for GTFS, given we'd need to programmatically examine the zip file to detect it (i.e., we couldn't extract the files otherwise they'd collide and overwrite one another).

Reply:
Please leave it as an ERROR. 

1. This is a de facto requirement met by thousands of feeds at Google.
2. If column names are duplicated, which column should we use then? How the feed provider may know that Validator uses a certain column? Consider ambiguous error messages that say that you have an invalid value in column `xyz` when there are multiple columns with that name.